harvester/harvest.py

```diff
@@ -515,7 +514,7 @@ def update_self_in_db(self) -> bool:
     def ckanify_dcatus(self) -> None:
         try:
             self.ckanified_metadata = ckanify_dcatus(
-                self.metadata, self.harvest_source.organization_id
+                self.metadata, self.harvest_source
```

Comment:
`self.harvest_source.organization_id` and `self.harvest_source` are different types, how does one replace the other? could it be `self.harvest_source.id`?

Reply:
Previously, we only required the org_id, but now we need additional fields from harvest_source. As a result, we are passing the entire object to allow access to the other fields from harvest_source.